packages/react-i18n/src/hooks.tsx

```diff
@@ -35,7 +35,7 @@ export function useI18n({
 
   React.useEffect(
     () => {
-      return manager.subscribe(ids, (translations, details) => {
+      manager.subscribe(ids, (translations, details) => {
```

Comment:
I think this needs to be returned, we might just need to rework the return value a bit. If we don't return this, it will never unsubscribe from the manager.

Reply:
👍 It was failing to compile because the return type was boolean.

Reply:
Fixed.